### PR TITLE
Added Supplier note attachment

### PR DESCRIPTION
**Pull request by make-it-git**
## Overview

| Q                 | A
| ----------------- | ---
| Bugfix?           | no
| New feature?      | yes
| Breaking Changes? | no
| Fixed issues      | #100 

Notify the following people: @darrynten

### CODE REVIEW

File: src/Models/SupplierNoteAttachment.php [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    /**
+     * Validates the specified Entity.
+     * @param string $id (not specified until implemented)
+     */
+    public function validate()
+    {
+        throw new LibraryException(
+            LibraryException::METHOD_NOT_IMPLEMENTED,
+            '\DarrynTen\SageOne\Models\SupplierNoteAttachment::validate()'
+        );
```
Comment (darrynten) [inline src/Models/SupplierNoteAttachment.php:129]: Do we know what this validate will eventually look like?

**Review by darrynten — COMMENTED** [review]
Small question

File: src/Models/SupplierNoteAttachment.php (hunk 1 above)
Comment (make-it-git) [reply src/Models/SupplierNoteAttachment.php:129]: #105

**Review by darrynten — APPROVED** [review]
😎

**Pull request merged**